scripts/gdpremail.php

```diff
@@ -0,0 +1,74 @@
+#!/usr/local/bin/php -q
+<?php
+/**
+ * DO NOT RUN THIS SCRIPT WILLY NILLY
+ * this script will email literally everyone that has not logged into myradio in over a year. That is a lot of people.
+ * after running this and waiting a couple weeks you should probably run the gdprdelete script
+ * 
+ * run with "php gdpremail.php"
+ * 
+ * Not actually tested for obvious reasons but probably works
+ */
+
```

Comment:
should these be a daemon, so they run automatically and in a controlled environment, say once a month rather than a script with a warning at the top of it

Reply:
This is a change that could definitely be made however i don't really want to put anything into a daemon until its been tested properly and I don't want to test it until it needs to be used because it will email A LOT of people.